test/lib/utils/lifecycle-cmd.js

```diff
@@ -20,6 +21,9 @@ t.test('create a lifecycle command', t => {
   cmd.exec(['some', 'args'], (er, result) => {
     t.same(runArgs, ['test-stage', 'some', 'args'])
     t.strictSame(result, 'called npm.commands.run')
-    t.end()
+  })
+  cmd.execWorkspaces(['some', 'args'], [], (er, result) => {
```

Comment:
Why do we care about this?

Reply:
otherwise these LOC are never covered in tests